QR-dtg.py

```diff
@@ -63,7 +59,7 @@ def main(webhook_url) -> None:
     opts.add_experimental_option('excludeSwitches', ['enable-logging'])
     opts.headless = True
     opts.add_argument('--log-level 3') 
-    driver = webdriver.Chrome(os.path.normpath(r"browser/chromedriver.exe"), options=opts)
+    driver = webdriver.Chrome(service=Service(os.path.normpath(r"browser/chromedriver.exe")), options=opts)
```

Comment:
executable_path has been deprecated this implies that the key executable_path will be deprecated in the upcoming releases.

This change is inline with the Selenium 4.0 Beta 1 [changelog](https://github.com/SeleniumHQ/selenium/blob/d6acda7c0254f9681574bf4078ff2001705bf940/py/CHANGES#L101) which mentions:

Deprecate all but Options and Service arguments in driver instantiation.